### PR TITLE
Ag inspection vc

### LINKED ISSUE #32: Redundancy with Schema.org
In creating new schemas for the AgInspectionReport, I noticed that in the current repo, we have schemas that are not modified or extended (as far as I can tell) from the ones they reference on GS1 or schema.org.  Why is that?  Shouldn't we simply ref out to Place, for example, rather than having it in the repo?

### PR DESCRIPTION
**Pull request by brianwmunz**
Updated the AgInspectionReport.test to create the VC credentials example.
While doing this I noticed that the Observation.json schema was throwing a context error due to the ID of measurement pointing to a non-existant schema.org schema.  Fixed this by pointing to the MeasuredValue.json local to the repo.  Also fixed a few other similar issues with contexts and naming.

After discussion in issue #32 I've taken all $ref referenced to external schemas on schema.org and brought those schemas locally, including only the properties needed for the schemas in the traceability repo. Also, this has required some general restructuring for existing schemas related to AgInspectionReport.json.
The new added schemas coming from schema.org are:

    Organization.json
    Person.json
    Product.json
    QuantitativeValue.json (for weight and/or size of Products)
    ParcelDelivery.json (for shipments of Products)

Also the Entity.json schema was created to describe an entity that could either be a Person or an Organization.

All testing was run and passed.

Fixes #32

### CODE REVIEW

File: docs/test-vectors/Entity.json
Comment (mprorock) [inline docs/test-vectors/Entity.json:18]: let's use example.org and/or example.com for all email addresses

**Review by mprorock — CHANGES_REQUESTED** [review]
let's remove apparent pi (event though it is not) such as example.org for all emails, and 555 numbers for all phone numbers

File: docs/test-vectors/Entity.json
Comment (brianwmunz) [reply docs/test-vectors/Entity.json]: Okay, changes made

**Review by mprorock — APPROVED** [review]

**Review by OR13 — APPROVED** [review]
This a truely excellent PR

**Comment by mprorock** [comment]
@mkhraisha any issues with merge?

**Comment by OR13** [comment]
I have left comments on https://github.com/w3c-ccg/traceability-vocab/pull/51 seems like they may share some significant amount of code. I am happy to merge incrementally as long as we open issues for any "requested changes"

File: docs/test-vectors/Entity.json
Comment (OR13) [reply docs/test-vectors/Entity.json]: see also https://github.com/w3c/did-core/pull/489#issuecomment-744544672

File: README.md
Comment (mprorock) [inline README.md:80]: Need this resolved or dropped before merge

**Review by mprorock — CHANGES_REQUESTED** [review]

**Comment by brianwmunz** [comment]
Mike and I decided it might be easiest to merge the changes from PR #51 into this one in order to ensure  there's no conflicts.  I've brought in the changes we made since submission of 51 so if we push this PR we should be all set with both and I can close 51.
Thanks.

**Comment by OR13** [comment]
unclear on the order of merges for this, deferring #51 and #52 to @mprorock , merge when ready

**Review by mprorock — APPROVED** [review]

**Pull request merged**
